app/src/main/res/xml/preferences.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+    <PreferenceCategory
+        android:title="@string/general_preference_category"
+        android:key="@string/general_preferences_key">
+        <Preference
+            android:title="@string/general_preference_notifications" />
+    </PreferenceCategory>
+    <PreferenceCategory
+        android:title="@string/preferences_fine_print"
+        android:key="@string/preferences_fine_print_key">
+        <Preference
+            android:title="@string/preferences_app_info" />
+        <Preference
+            android:title="@string/preferences_terms_of_service" />
+        <Preference
+            android:title="@string/preferences_privacy" />
+        <Preference
+            android:title="@string/preferences_licences">
+            <intent
+                android:action="android.intent.action.VIEW"
```

Comment:
The following change will explicitly open our license view activity, skipping the "pick your app" bottom modal:
```diff
diff --git a/app/src/main/res/xml/preferences.xml b/app/src/main/res/xml/preferences.xml
index 9df9665..5ec74e9 100644
--- a/app/src/main/res/xml/preferences.xml
+++ b/app/src/main/res/xml/preferences.xml
@@ -19,7 +19,8 @@
             android:title="@string/preferences_licences">
             <intent
                 android:action="android.intent.action.VIEW"
-                android:targetClass="OssLicensesMenuActivity.class" />
+                android:targetPackage="org.gophillygo.app"
+                android:targetClass="com.google.android.gms.oss.licenses.OssLicensesMenuActivity" />
         </Preference>
     </PreferenceCategory>
```

Reply:
Excellent, I was wondering how to do that.

Reply:
Note from the diff that you need to give the full path to the `targetClass` as well - since it has a different package than the application.